client/src/Components/AddWorkButton/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import {
+  message, Modal, Input, Upload, Button, Form,
+} from 'antd';
+import { useTranslation } from 'react-i18next';
+import { UploadOutlined } from '@ant-design/icons';
+import { request } from '../../utils';
+
+function AddWorkButton() {
+  const { t } = useTranslation();
+  const [isClickAddWork, setIsClickAddWork] = useState(false);
+  const [form] = Form.useForm();
+  const { TextArea } = Input;
+
+  const handelAddWork = () => {
+    try {
+      form
+        .validateFields()
+        .then(async (values) => {
+          const { title, content, image } = values;
+          await request('post', '/work', {
+            title, content, image: image[0],
+          });
+
+          setIsClickAddWork(false);
+          message.success(t('successfully-added'));
+        });
+    } catch (error: any) {
+      message.error(t('error-delete-message'));
+    }
+  };
+  const normFile = (e: any) => {
+    if (Array.isArray(e)) return e;
+    return e && e.fileList;
+  };
+  const checkImage = (file: any) => {
```

Comment:
try to user accept props to Upload Component to allow to user upload specific files type 
in ant design: 
![image](https://user-images.githubusercontent.com/54906217/168445337-fc8c037c-97cb-461b-acc7-b5a019285570.png)

and if you jump to See [input accept Attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept)

`<input type="file" id="docpicker"
  accept=".doc,.docx,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document">`